app/command_mute.go

```diff
@@ -43,14 +43,20 @@ func (me *MuteProvider) DoCommand(a *App, args *model.CommandArgs, message strin
 		return &model.CommandResponse{Text: args.T("api.command_mute.error", map[string]interface{}{"Channel": channel.DisplayName}), ResponseType: model.COMMAND_RESPONSE_TYPE_EPHEMERAL}
 	}
 
+	channelName := ""
+	splitMessage := strings.Split(message, " ")
 	// Overwrite channel with channel-handle if set
 	if strings.HasPrefix(message, "~") {
-		splitMessage := strings.Split(message, " ")
-		chanHandle := strings.Split(splitMessage[0], "~")[1]
-		data := (<-a.Srv.Store.Channel().GetByName(channel.TeamId, chanHandle, true)).Data
+		channelName = strings.Split(splitMessage[0], "~")[1]
```

Comment:
Why split? You already checked the prefix so you can just do `splitMessage[0][1:]`

Reply:
done